broadcast/src/linked/signer/actor.rs

```diff
@@ -495,7 +540,31 @@ impl<
             height: chunk.height,
         };
         let digest = D::try_from(&chunk.payload).unwrap();
-        self.application.verify(context, digest).await;
+
+        // Capture needed variables for our async verification future.
+        let verify_timeout = self.verify_timeout;
```

Comment:
I suggest we avoid the timeout and instead just bound the backlog to the application (causing us to stop making requests if we fall behind).

In gnarly situations, I've found timeouts actually prevent us from making progress (as we end up timing out work before the application gets to it, over and over again).